tools/github_actions_install.sh

```diff
@@ -8,12 +8,8 @@ if [[ "$SPHINX_VERSION" == "" ]]; then
     SPHINX_INSTALL=""
 elif [[ "$SPHINX_VERSION" == "dev" ]]; then
     SPHINX_INSTALL="git+https://github.com/sphinx-doc/sphinx"
-    if [[ "$1" == "doc" ]]; then
-        # Until they release a new version that undoes the max sphinx pin...
-        DEP_EXTRA="git+https://github.com/executablebooks/MyST-NB"
-    fi
 elif [[ "$SPHINX_VERSION" == "old" ]]; then
-    SPHINX_INSTALL="sphinx==5.0"
+    SPHINX_INSTALL="sphinx==6.1.0"
```

Comment:
Is there a reason this is pinned to `6.1.0` rather than the newer `6.2.1`?

See ref: https://pypi.org/project/Sphinx/6.2.1/

Reply:
our minimal requirement is coming from autoapi: sphinx>=6.1.0 so I set myself on this lowest pin.